discovery-provider/src/utils/ipfs_lib.py

```diff
@@ -164,12 +164,21 @@ def cat(self, multihash):
             logger.error(f"IPFSCLIENT | IPFS cat timed out for CID {multihash}")
             raise  # error is of type ipfshttpclient.exceptions.TimeoutError
 
-    def multihash_is_directory(self, multihash):
+    def multihash_is_directory(self, multihash, is_square=True):
+        """Given a profile picture or cover photo CID, determine if it's a
+        directory or a regular file CID
+
+        self - class self
```

Comment:
super nit, we should standardize how we document params in docstrings. i think there are a few versions we use in discprov. we already have

```
"""
    Returns the usage metrics for routes

    Args:
        args: dict THe parsed args from the request
        args.path: string The route path of the query
        args.start_time: date The start of the query
        args.query_string: optional string The query string to filter on
        args.limit: number The max number of responses to return
```

as well as

```
:param args: dictionary
    :param args.verbose: bool
        if True, returns db connection information
    :param args.healthy_block_diff: int
        determines the point at which a block difference is considered unhealthy
    :param args.enforce_block_diff: bool
        if true and the block difference is unhealthy an error is returned

    :rtype: (dictionary, bool)
    :return: tuple of health results and a boolean indicating an error
```

Reply:
this is a fair point. so i personally use pep 257 standard docstrings https://www.python.org/dev/peps/pep-0257/ but we should make everything consistent

Reply:
afaict, pep 257 doesn't really specify how exactly to do params
https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings
and i know google's style guide is different, etc. i'm good w. any of the above :)

Reply:
i used the first version above. I think it's similar to the google style guide which i personally like better